app/services/sufia/statistics/collections.rb

```diff
@@ -0,0 +1,31 @@
+module Sufia
+  module Statistics
+    class Collections
+      # @param [Fixnum] delta_x change in x (in days)
+      # @param [Date] x_min minimum date
+      # @param [Date] x_max max date
+      def initialize(delta_x: 7, x_min: 1.month.ago.beginning_of_day, x_max: Date.today.end_of_day)
+        @delta_x = delta_x
+        @x_min = x_min
+        @x_max = x_max
+      end
+
+      def points
```

Comment:
Would it be useful to make this an enumerable? At the very least, it'd read better to me.

```ruby
def points
  return to_enum(:points) unless block_given?

  x = @x_min
  while (x <= @x_max)
    x += @delta_x.days
    yield [ x.to_i * 1000, y: collections_in_range(@x_min, x) ]
  end 
```